content/docs/hooks-faq.md

```diff
@@ -18,126 +18,126 @@ This page answers some of the frequently asked questions about [Hooks](/docs/hoo
-  * [What can I do if my effect dependencies change too often?](#what-can-i-do-if-my-effect-dependencies-change-too-often)
-  * [How do I implement shouldComponentUpdate?](#how-do-i-implement-shouldcomponentupdate)
-  * [How to memoize calculations?](#how-to-memoize-calculations)
-  * [How to create expensive objects lazily?](#how-to-create-expensive-objects-lazily)
-  * [Are Hooks slow because of creating functions in render?](#are-hooks-slow-because-of-creating-functions-in-render)
-  * [How to avoid passing callbacks down?](#how-to-avoid-passing-callbacks-down)
-  * [How to read an often-changing value from useCallback?](#how-to-read-an-often-changing-value-from-usecallback)
-* **[Under the Hood](#under-the-hood)**
-  * [How does React associate Hook calls with components?](#how-does-react-associate-hook-calls-with-components)
-  * [What is the prior art for Hooks?](#what-is-the-prior-art-for-hooks)
-
-## Adoption Strategy {#adoption-strategy}
-
-### Which versions of React include Hooks? {#which-versions-of-react-include-hooks}
-
-Starting with 16.8.0, React includes a stable implementation of React Hooks for:
+* **[Strategia adopcji](#adoption-strategy)**
+  * [Które wersje Reacta wspierają hooki?](#which-versions-of-react-include-hooks)
+  * [Czy muszę przepisać wszystkie komponenty klasowe?](#do-i-need-to-rewrite-all-my-class-components)
+  * [Co mogę zrobić z hookami czego nie mogłem zrobić z klasami?](#what-can-i-do-with-hooks-that-i-couldnt-with-classes)
+  * [Jaka część mojej wiedzy o Reakcie jest nadal aktualna?](#how-much-of-my-react-knowledge-stays-relevant)
+  * [Czego mam używać: hooków, klas, a może mieszać oba sposoby?](#should-i-use-hooks-classes-or-a-mix-of-both)
+  * [Czy hooki pokrywają wszystkie przypadki użycia, które są dostępne dla klas?](#do-hooks-cover-all-use-cases-for-classes)
+  * [Czy hooki zastępują właściwości renderujące i komponenty wyższego rzędu?](#do-hooks-replace-render-props-and-higher-order-components)
+  * [Co hooki oznaczają dla popularnych API takich jak connect() z Reduxa lub React Router?](#what-do-hooks-mean-for-popular-apis-like-redux-connect-and-react-router)
+  * [Czy hooki współpracują ze statycznym typowaniem?](#do-hooks-work-with-static-typing)
+  * [Jak testować komponenty, które używają hooków?](#how-to-test-components-that-use-hooks)
+  * [Czego dokładnie wymagają reguły lintera?](#what-exactly-do-the-lint-rules-enforce)
+* **[Od klas do hooków](#from-classes-to-hooks)**
+  * [Jak wyglądają metody cyklu życia w odniesieniu do hooków?](#how-do-lifecycle-methods-correspond-to-hooks)
+  * [Jak mogę pobrać dane wykorzystując hooki?](#how-can-i-do-data-fetching-with-hooks)
+  * [Czy istnieje coś podobnego do zmiennych instancji?](#is-there-something-like-instance-variables)
+  * [Lepiej używać jednego czy wielu zmiennych stanu?](#should-i-use-one-or-many-state-variables)
+  * [Czy mogę uruchomić efekt tylko podczas aktualizacji?](#can-i-run-an-effect-only-on-updates)
```

Comment:
Coś nie do końca pasuje mi tutaj `aktualizacja`, ale samemu mi trudno znaleźć lepszy wyraz. Może po prostu zapisać to jako `Czy mogę uruchomić efekt tylko podczas nowych zmian?`

Reply:
Co myślisz o: `Czy mogę pominąć efekt w przypadku nowych zmian?`

Reply:
Jestem za dopisaniem "komponentu".

Reply:
Dzięki @idbartosz  i @jakubdrozdek . Wrzuciłem tłumaczenie z dopiskiem "komponentu" -> `Czy mogę pominąć efekt podczas aktualizacji komponentu?`. 👍

Reply:
@bartlomiejzuber Nie widzę słowa "komponentu" - na pewno wrzuciłeś? ;)